CorePayments/src/main/java/com/paypal/android/corepayments/apis/eligibility/EligibilityAPI.kt

```diff
@@ -0,0 +1,61 @@
+package com.paypal.android.corepayments.apis.eligibility
+
+import android.content.Context
+import com.paypal.android.corepayments.CoreConfig
+import com.paypal.android.corepayments.PayPalSDKError
+import com.paypal.android.corepayments.R
+import com.paypal.android.corepayments.ResourceLoader
+import com.paypal.android.corepayments.features.eligibility.EligibilityRequest
+import com.paypal.android.corepayments.graphql.GraphQLClient
+import com.paypal.android.corepayments.optBooleanAtKeyPath
+import org.json.JSONArray
+import org.json.JSONObject
+
+internal class EligibilityAPI(
+    private val config: CoreConfig,
+    private val graphQLClient: GraphQLClient = GraphQLClient(config),
+    private val resourceLoader: ResourceLoader = ResourceLoader()
+) {
+    companion object {
+        const val VARIABLE_CLIENT_ID = "clientId"
+        const val VARIABLE_INTENT = "intent"
+        const val VARIABLE_CURRENCY = "currency"
+        const val VARIABLE_ENABLE_FUNDING = "enableFunding"
+    }
+
+    @Throws(PayPalSDKError::class)
+    suspend fun checkEligibility(context: Context, request: EligibilityRequest): Eligibility {
+        val query = resourceLoader.loadRawResource(context, R.raw.graphql_query_funding_eligibility)
+        val enableFundingMethods = listOf(SupportedPaymentMethod.VENMO.name)
```

Comment:
What is this?

Reply:
For the most part it's equivalent to [this line](https://github.com/paypal/paypal-ios/blob/4335a57dab36d89b21257f617ffb02265aa0c712/Sources/CorePayments/Eligibility/EligibilityRequest.swift#L26) in `EligibilityRequest.swift`.

Reply:
Doesn't it just return "VENMO"?

Reply:
I'm just curious, why do we need this when the endpoint seems to return all eligibility for all the payment methods?

Reply:
Yeah that's a good question actually I'm not sure. The GraphQL query does seem like it should take a dynamic array of requested payment method types.